ts/packages/node/src/stores/AWSParameterStore.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  SSMClient,
+  SSMClientConfig,
+  DeleteParameterCommand,
+  GetParameterCommand,
+  PutParameterCommand,
+  ParameterType,
+} from '@aws-sdk/client-ssm';
+import { KeyValueConfigStore } from '@configu/ts';
+
+export type AWSParameterStoreConfigStoreConfiguration = SSMClientConfig;
+
+export class AWSParameterStoreConfigStore extends KeyValueConfigStore {
+  private client: SSMClient;
+  constructor(configuration: AWSParameterStoreConfigStoreConfiguration) {
+    super('aws-parameter-store');
+    this.client = new SSMClient(configuration);
+  }
+
+  // * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ssm/command/GetParameterCommand/
+  async getByKey(key: string): Promise<string> {
+    const command = new GetParameterCommand({ Name: key });
+    const res = await this.client.send(command);
+    return res?.Parameter?.Value ?? '';
+  }
+
+  // * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/ssm/command/PutParameterCommand/
+  async upsert(key: string, value: string): Promise<void> {
+    const command = new PutParameterCommand({ Name: key, Value: value, Type: ParameterType.STRING, Overwrite: true });
```

Comment:
I would recommend `ParameterType.SECURE_STRING` here - since a big portion of Configu's use case entails storing things like DB connections/passwords/other highly sensitive data, a lot of users will require these to be encrypted at rest.

Going down this route means you'll need to add `WithDecryption: true` to your `GetParameterCommand` too.